libs/composer/composer.json

```diff
@@ -0,0 +1,18 @@
+{
+	"name": "ilias/deps",
+	"description": "Dependencies of ILIAS",
+	"license": "GPL v3",
+	"authors": [],
+	"require": {
+		"filp/whoops": "^2.0"
+	},
+	"extra": {
+		"filp/whoops" : {
+			"added_by" : "Denis Klöpfer <denis.kloepfer@concepts-and-training.de>",
```

Comment:
Hi Denis,

"source" is missing here. Please also reorder the fields to source, version, wrapped, added, update.

Thx!